src/classBased/components/InputTodo.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  state = {
+    title: '',
+  };
+
+  onChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    if (this.state.title.trim()) {
+      this.props.addTodoProps(this.state.title);
+      this.setState({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
```

Comment:
[REQUIRED]:Please avoid using window.alert() . It's impossible to style and displays differently on different browsers. It would be best to customize the error message using CSS.